Jenkinsfile

```
@@ -120,7 +120,7 @@ pipeline {
           steps {
             catchError(stageResult: 'FAILURE') {
               withMaven(jdk: 'jdk-8-latest', maven: 'maven-3.2-latest', mavenSettingsConfig: 'camunda-maven-settings', options: [artifactsPublisher(disabled: true), junitPublisher(disabled: true)]) {
-                runMaven(true, false, false, cambpmGetMavenTargetDir('engine-unit'), cambpmGetMavenTargetAndProfile('engine-unit') + cambpmGetDbProfiles('h2'))
+                cambpmRunMavenByStageType('engine-unit', 'h2', params.STANDALONE)
```

The STANDALONE param is not defined in main CE pipe, other put a define value to false or define the parameter.
The ci is failing with 
```
groovy.lang.MissingMethodException: No signature of method: cambpmRunMavenByStageType.call() is applicable for argument types: (java.lang.String, java.lang.String, null) values: [engine-unit, h2, null]

Possible solutions: call(java.lang.String, java.lang.String, boolean), call(java.lang.String, java.lang.String, boolean, boolean), wait(), any(), run(), run()
```
```suggestion
                cambpmRunMavenByStageType('engine-unit', 'h2', false)
```

---

Fixed by directly referencing the `params.STANDALONE` parameter, as we do in the `with Labels` method.